src/jdk.incubator.foreign/share/classes/jdk/internal/foreign/Utils.java

```diff
@@ -52,7 +52,7 @@
     private static final String foreignRestrictedAccess = Optional.ofNullable(VM.getSavedProperty("foreign.restricted"))
             .orElse("deny");
 
-    private static final MethodHandle SEGMENT_FILTER;
+    public static final MethodHandle SEGMENT_FILTER;
```

Comment:
Why this change?

Reply:
this is a leftover from other experiments - will remove